src/char/pincode.c

```diff
@@ -185,11 +185,13 @@ void pincode_setnew(int fd, struct char_session_data* sd)
 	if (pincode->check_blacklist && pincode->isBlacklisted(newpin)) {
 		pincode->makestate(fd, sd, PINCODE_MAKE_RESTRICT_PW);
 		return;
+	} else {
```

Comment:
Else branch here useless, because inside if was return. if return executes, other code in else or outside of if will be never called.
from actual changes i see here only like 194

Reply:
Fixed.